liferay-echo/liferay-portal
# LPS-196155 Improvements for Page Audit

Motivation
=======
[Link to story](https://liferay.atlassian.net/browse/LPS-196044)
[Link to subtask](https://liferay.atlassian.net/browse/LPS-196155)

Steps to Verify:
----------------
1. Create a manual collection
1. In Documents and Media add a big image (>500kb)
1. Go to home page and add an image fragment with the image uploaded in step 2
2. In the image fragment configuration select resolution: auto 
![Captura de pantalla 2023-09-20 a las 12 26 01](https://github.com/liferay-echo/liferay-portal/assets/3276218/dd1bec7f-f5dc-4f48-95f5-c0b6ec457db1)

4. Check that the lazy load configuration is above “Image description” field
5. Add a collection display fragment and display the collection created in step 1 in this fragment
6. In the collection display fragment configuration check the option "Display All Collection Items"
1. Publish the page and open the Page Audit panel
1. Check that the fragment list spacing is correct
2. Click on the collection display fragment and check that the warning message is correct
3. Click on the image fragment and check that the warning message is correct
4. Check that the "All" option is available in the page audit filters
5. Go to Design-> Fragments and check that the label "CACHED" color is correct.
